### PR TITLE
Update approval-workflow.yml

### LINKED ISSUE #50: Approval Workflow not working
Pull requests are not getting labelled as `approved` after two approvals, Actions shows an error
```json
{
  "message": "Resource not accessible by integration",
  "documentation_url": "https://docs.github.com/rest/reference/issues#add-labels-to-an-issue"
}
```

### Expected Behaviour
Pull requests should be labelled as `approved` after 2 approval

If you are working on this, also change label to `hacktoberfest-approved`

### PR DESCRIPTION
**Pull request by Victoria-DR**
Change workflow file being run

Closes #50

### CODE REVIEW

**Comment by Victoria-DR** [comment]
Looks like `pullreminders` was discontinued and the repo owner is now `abinoda` but they didn't update their documentation.

File: .github/workflows/approval-workflow.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Label when approved
-      uses: pullreminders/label-when-approved-action@master
+      uses: abinoda/label-when-approved-action@1.0.5
       env:
         APPROVALS: "2"
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         ADD_LABEL: "approved"
```
Comment (priyansh32) [inline .github/workflows/approval-workflow.yml:12]: Can you change label to `hacktoberfest-approved`

**Review by priyansh32 — CHANGES_REQUESTED** [review]

**Review by priyansh32 — APPROVED** [review]

**Review by roshgupta — APPROVED** [review]

**Comment by priyansh32** [comment]
@Victoria-DR It's still throwing same error after two approvals

**Comment by Victoria-DR** [comment]
I did a bunch of testing on a new repo and got a different error, but still couldn't get it to pass successfully. Unfortunately, I can't figure it out and hopefully someone else will come around and be able to help. I'll be closing this PR and best of luck!

**Comment by Victoria-DR** [comment]
Huh, looks like it works now. Not sure why though. 🤔

**Comment by priyansh32** [comment]
Thanks for the efforts @Victoria-DR. 
I have added another app for this, It [trafico bot](https://github.com/marketplace/trafico-pull-request-labeler) that labelled the PR

**Pull request closed without merging**